Remove the object tracking code as it is causing memory leaks
in sanitizer builds, https://crbug.com/908138.
This code is already removed from upstream chromium //base code.

diff --git a/base/tracked_objects.cc b/base/tracked_objects.cc
index 1507c09..882f5fb 100644
--- a/base/tracked_objects.cc
+++ b/base/tracked_objects.cc
@@ -891,7 +891,6 @@ void ThreadData::ShutdownSingleThreadedCleanup(bool leak) {
 // static
 ThreadData* ThreadData::GetRetiredOrCreateThreadData(
     const std::string& sanitized_thread_name) {
-  SCOPED_UMA_HISTOGRAM_TIMER("TrackedObjects.GetRetiredOrCreateThreadData");
 
   {
     base::AutoLock lock(*list_lock_.Pointer());
